.github/workflows/e2e.sign-attestations.schedule.yml

```diff
@@ -44,13 +44,13 @@ jobs:
         with:
           node-version: 16
       - name: install sigstore-js
-        run: npm ci
+        run: npm install -g @sigstore/cli@^0.8.0
```

Comment:
Does npm have a concept of `latest`? Then you don't have to worry about updating to the latest sigstore-js CLI

Reply:
Yes we could always install the latest
```
npm install -g @sigstore/cli
```
, though we generally like to pin versions when we can. But I guess if the signer's version is already pinned, it could be okay to leave the verifier unpinned. @ianlewis wdyk?

Reply:
Let's keep pinning, this is useful for determinism.  A better way to do to use a lock file, so that dependabot / renovatebot sends us PR. renovatebot supports custom regex to let it know it should interpret this `^0.8.0` as a version to update. I think it's easier to use a lock file to pin the dep.

Reply:
Any reason why we don't pin in the `package.json` in the project root? I managed the versions for dev dependencies there so that renovate could send PRs to update them etc.

https://github.com/slsa-framework/slsa-github-generator/blob/main/package.json

Reply:
Thanks, that's a better place for this.